libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -415,8 +457,10 @@ class ctx final : public virtual context {
 
 private:
   map_type context_entries;
+  detail::subnet_tree subnet_entries;
 };
 
+// TODO: do we need a v2 loader now with the subnet optimization?
```

Comment:
@tenzir/pipelines-squad what about this? 👆

Reply:
Same as above—if you avoid persisting the patricia trie in the first place this is irrelevant.